lib/worker.js

```diff
@@ -53,3 +56,42 @@ if (!module.parent) {
   // HTTPS
   if (config.ssl.listen !== false) https.createServer(config.ssl, app).listen(config.ssl.listen || 8443);
 }
+
+// And start a cleanup loop over the work directory; all workers will run the same loop which is ok - deleting
+// orphaned in-flight work is idempotent and we'll
```

Comment:
seems like the comment wasn't quite finished. Comment readers everywhere will be left hanging on the edge of their seat.